### ISSUE TITLE
How can i make the theme change colors with vivaldifox?

### ISSUE THREAD

**Issue by BigLadDylan** [KiKaraage/ArcWTF#32]
https://addons.mozilla.org/en-GB/firefox/addon/vivaldifox/

Thanks for the cool theme btw.

**Comment by KiKaraage** [comment]
It's possible... kinda, but seems like Vivaldifox doesn't supported dynamic theme mode change (example: When I use light mode by default, once I visited a website with dark mode, I would love to use light toolbar icon, light URL text, and such.) Because ArcWTF uses transparent URLbar and use the same color (--lwt-accent-color) for the whole navbar + tabbar + #browser-container border, the coloring might clashed when you visited dark websites in light mode. Anyway here's a config that worked if you're okay with that condition:

![gambar](https://github.com/KiKaraage/ArcWTF/assets/10529881/6f54bd40-3e8e-4e33-81a9-a27c69257c62)
![gambar](https://github.com/KiKaraage/ArcWTF/assets/10529881/654ab04e-27c5-44b9-86f2-0ceca9e5d86e)

I advise you to use Adaptive Tab Bar Color instead, the automatic color scheme fits ArcWTF very well 👌 Here's my setup:
![gambar](https://github.com/KiKaraage/ArcWTF/assets/10529881/faffaf6d-0a86-4eb4-81c4-5c0e759ac62d)
![gambar](https://github.com/KiKaraage/ArcWTF/assets/10529881/fcc0ac0d-852e-48b3-bd19-0d479fc7448f)

**Comment by BigLadDylan** [comment]
Ive been experimenting with https://github.com/Frewacom/pywalfox and ive managed to get it working i think?

I created [spotipaper](https://github.com/BigLadDylan/spotipaper) that would work amazingly with this theme +  pywalfox.

But yeah thanks for the feedback on vivaldi I really appreciate it, I will check that out if i ever get bored of pywal.

**Comment by KiKaraage** [comment]
Two steps:

1. Changing toolbar icons scale [on toolbar/navbar.css line 13](https://github.com/KiKaraage/ArcWTF/blob/bb6f2b7ef7e3d201e23d86bf8636e5d0ea4bd68b/toolbar/navbar.css#L13). Default is 0.85, you can change it to 1 or 2.
2. Changing URL bar height. The original code are Here are the original code from [toolbar/urlbar.css line 17](https://github.com/KiKaraage/ArcWTF/blob/bb6f2b7ef7e3d201e23d86bf8636e5d0ea4bd68b/toolbar/urlbar.css#L17), feel free to modify:

```
 /* urlbar height */
  & #urlbar-container {
    --urlbar-container-height: calc(8px + 16px + var(--toolbarbutton-inner-padding) * 2) !important;
  }

  & #urlbar {
    --urlbar-toolbar-height: calc(5px + 16px + var(--toolbarbutton-inner-padding) * 2) !important;
    --urlbar-height: calc(16px + var(--toolbarbutton-inner-padding) * 2) !important;
    --urlbar-min-height: calc(16px + var(--toolbarbutton-inner-padding) * 2) !important;
    font-size: 1em !important;
  }
```

**Comment by BigLadDylan** [comment]
yeah i played with that but i could only seem to move the url bar... i would just like a bit more padding on the toolbar because its quite squished atm
![image](https://github.com/KiKaraage/ArcWTF/assets/76881322/6079fbdd-0fc5-4deb-a0c8-5ee6a49bc8f7)

**Comment by KiKaraage** [comment]
Oh you should edit [toolbar/navbar.css line 8](https://github.com/KiKaraage/ArcWTF/blob/bb6f2b7ef7e3d201e23d86bf8636e5d0ea4bd68b/toolbar/navbar.css#L8) then, especially `--toolbarbutton-outer-padding`. Sorry for forgetting it 😅

```
:root {
  --toolbarbutton-border-radius: 4px !important;
  --toolbarbutton-inner-padding: 4px !important;
  --uc-toolbarbutton-inner-inline-padding: 4px !important;
  --toolbarbutton-outer-padding: 6px !important;
  --tabs-navbar-shadow-size: 0px !important;
  --message-bar-border-radius: var(--border-radius-medium);
}
```
